server/app/controllers/admin/AdminProgramBlockPredicatesController.java

```diff
@@ -434,11 +441,26 @@ public Result updateEligibilityMessage(Request request, long programId, long blo
 
   /** HTMX partial that renders a card for editing a condition within a predicate. */
   @Secure(authorizers = Authorizers.Labels.CIVIFORM_ADMIN)
-  public Result hxEditCondition(Request request) {
-    DynamicForm formData = formFactory.form().bindFromRequest(request);
-    long conditionId = Long.parseLong(formData.get("conditionId"));
-    return ok(editConditionPartialView.render(
-            request, new EditConditionPartialViewModel(conditionId)))
-        .as(Http.MimeTypes.HTML);
+  public Result hxEditCondition(
+      Request request, long programId, long blockDefinitionId, String predicateUseCaseString) {
```

Comment:
(blocking) predicateUseCaseString needs to match the naming used in the routes file.